src/screens/FeatureList.js

```diff
@@ -7,13 +7,28 @@
 
 import React from 'react';
 import Features from '../components/feature_activation/Features';
+import { useNewUiEnabled } from '../hooks';
```

Comment:
These changes are also part of the other PR

Reply:
PR merged with master. Waiting Review.🙂